db/dcrpg/pgblockchain.go

```diff
@@ -33,6 +34,27 @@ import (
 	humanize "github.com/dustin/go-humanize"
 )
 
+const (
+	// InitialAgendaState is the agenda status when the agenda is up for voting
+	// but the votes tally is not available.
+	InitialAgendaState = "defined"
+
+	// FailedAgendaState is the agenda state set when the votes tally does not
+	// attain the minimum threshold set. Activation height is not set for such an
+	// agenda.
+	FailedAgendaState = "failed"
+
+	// PassedAgendaState is the agenda state when the agenda is considered to
+	// have passed after attaining the minimum set threshold. This agenda can
+	// it activation height set.
+	PassedAgendaState = "lockedin"
+
+	// ActivatedAgendaState is the agenda state set after 8064 blocks since when
```

Comment:
I suggest referring to `chaincfg.RuleChangeActivationInterval`(e.g. 8064 blocks = 2016 * 4 for 4 weeks on mainnet).